### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getCursors' of undefined

### ISSUE THREAD

**Issue by tharris-fps** [nikhilkalige/docblockr#135]
[Enter steps to reproduce below:]
1. Creating a watchpoint

**Atom Version**: 1.0.19
**System**: Linux Mint
**Thrown From**: [docblockr](https://github.com/nikhilkalige/docblockr) package, v0.7.3
### Stack Trace

Uncaught TypeError: Cannot read property 'getCursors' of undefined

```
At /home/tharris/.atom/packages/docblockr/lib/docblockr-worker.js:200

TypeError: Cannot read property 'getCursors' of undefined
  at DocBlockrAtom.module.exports.DocBlockrAtom.validate_request (/home/tharris/.atom/packages/docblockr/lib/docblockr-worker.js:200:36)
  at atom-workspace.<anonymous> (/home/tharris/.atom/packages/docblockr/lib/docblockr-worker.js:73:17)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:245:29)
  at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:524:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:347:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app.asar/src/window-event-handler.js:184:20)

```
### Commands

```
     -5:10.9.0 blur (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -5:08.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -5:08.6.0 blur (atom-text-editor.editor.mini.is-focused)
     -5:08.6.0 php-debug:run (atom-text-editor.editor.vim-mode.normal-mode)
     -5:06.8.0 core:save (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
  6x -5:03.3.0 blur (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -5:01.2.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -5:01.2.0 core:cancel (atom-text-editor.editor.mini.is-focused)
undefined -5:01.1.0 blur (atom-text-editor.editor.mini.is-focused)
     -2:22.6.0 command-palette:toggle (atom-text-editor.editor.vim-mode.normal-mode)
     -2:22.4.0 blur (atom-text-editor.editor.vim-mode.normal-mode)
     -2:19.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:19.8.0 blur (atom-text-editor.editor.mini.is-focused)
     -2:19.8.0 settings-view:open (atom-text-editor.editor.vim-mode.normal-mode)
 51x -2:19.3.0 blur (atom-text-editor.editor.vim-mode.normal-mode)
     -0:01.9.0 docblockr:parse-enter (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ],
    "autoHideMenuBar": true,
    "disabledPackages": [
      "linter-phpcs"
    ]
  },
  "docblockr": {}
}
```
### Installed Packages

``` coffee
# User
atom-beautify, v0.28.15
atom-material-syntax, v0.2.4
atom-material-ui, v0.6.2
autocomplete-php, v0.3.7
docblockr, v0.7.3
drupal, v1.8.0
file-icons, v1.6.11
linter, v1.9.1
linter-php, v1.0.5
minimap, v4.15.0
php-debug, v0.1.3
php-twig, v4.0.0
pigments, v0.18.0
project-manager, v2.6.5
todo-show, v0.16.0
vim-mode, v0.61.0

# Dev
No dev packages
```

**Comment by miclf** [comment]
I got the exact same thing when trying to create a newline (pressing the enter key) in the *Documentation* tab of the *Details* panel.

**Comment by miclf** [comment]
Additional note to the previous comment: this happens when using the API Workbench package. Looks like a conflict.

**Comment by rawkode** [comment]
Got this error with API Workbench also

**Comment by abhi18av** [comment]
Yup, still have the error. Trying to run the idea-haskell package repl and this red-banner shows up.

**Comment by orosznyet** [comment]
Got this in API Workbench. Came here to join the train.

```
TypeError: Cannot read property 'getCursors' of undefined
    at DocBlockrAtom.module.exports.DocBlockrAtom.validate_request (C:\Users\ONet\.atom\packages\docblockr\lib\docblockr-worker.js:208:36)
    at atom-workspace.<anonymous> (C:\Users\ONet\.atom\packages\docblockr\lib\docblockr-worker.js:75:17)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\ONet\AppData\Local\atom\app-1.10.2\resources\app.asar\src\command-registry.js:260:29)
    at C:\Users\ONet\AppData\Local\atom\app-1.10.2\resources\app.asar\src\command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\ONet\AppData\Local\atom\app-1.10.2\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\ONet\AppData\Local\atom\app-1.10.2\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (C:\Users\ONet\AppData\Local\atom\app-1.10.2\resources\app.asar\src\window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (C:\Users\ONet\AppData\Local\atom\app-1.10.2\resources\app.asar\src\window-event-handler.js:3:61
```

**Comment by mrstux** [comment]
ended up typing this line in due to a copy past error

`		kind:'electron',	// phantomjs or electron			kind:'electron',	// phantomjs or electron`

was trying to put a return before the second kind... and I can't because docblockr keeps on throwing this error.

**Comment by perryprog** [comment]
Same problem ^^^

**Comment by perryprog** [comment]
@lierdakil made some helpful points. Would like to see this fixed as it prevents me from using a Haskell REPL.

**Comment by Arcanemagus** [comment]
BTW, this breaks the commit message box of the new GitHub integration coming up in Atom in the next beta.

**Comment by perryprog** [comment]
Yay!

**Comment by MoritzKn** [comment]
Also landed in v0.9.5 btw 🚢
